net-misc/tigervnc/tigervnc-1.10.0.ebuild

```diff
@@ -0,0 +1,200 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
I believe EAPI-7 should work here, if you switched from `gnome2_icon_cache_update` to `xdg_icon_cache_update` as you already inherit *xdg-utils*. You can then drop *gnome2-utils.eclass* as it doesn't seem to be used for anything else.